### PR TITLE
Discount choice cards

### PR DESCRIPTION
**Pull request by charleycampbell**
## What does this change?
Adds logic to calculate and display new discounted amounts for Support+ on the three tier choice cards

## Why?
Acquisitions have been a bit low, so we are hoping we can attract more people to sign up to Support+ at a discounted amount

https://trello.com/c/pB51sDky/718-discount-on-epic-choice-cards

## Screenshots
image

### CODE REVIEW

File: dotcom-rendering/src/components/marketing/epics/ThreeTierChoiceCardData.tsx [hunk 1]
```diff
@@ -13,8 +14,28 @@ export const ChoiceCardTestData_REGULAR: ChoiceInfo[] = [
...
-			`Support ${currencySymbol}${amount}/month`,
+		label: (
+			amount: number,
+			currencySymbol: string,
+			discount?: number,
+		): JSX.Element | string => {
+			if (!isUndefined(discount)) {
+				return (
+					<>
+						{' '}
```
Comment (tomrf1) [inline dotcom-rendering/src/components/marketing/epics/ThreeTierChoiceCardData.tsx:25]: do we need this space here?

File: dotcom-rendering/src/components/marketing/epics/ThreeTierChoiceCardData.tsx [hunk 2]
```diff
@@ -33,6 +54,8 @@ export const ChoiceCardTestData_REGULAR: ChoiceInfo[] = [
 	},
 ];
 
+//new discount set of choice cards and pass through the choice card variant
```
Comment (tomrf1) [inline dotcom-rendering/src/components/marketing/epics/ThreeTierChoiceCardData.tsx:57]: can be removed

File: dotcom-rendering/src/components/marketing/epics/ThreeTierChoiceCards.tsx [hunk 3]
```diff
@@ -143,11 +166,16 @@ const RecommendedPill = () => {
 	return <div css={recommendedPillStyles}>Recommended</div>;
 };
 
+const DiscountedPill = () => {
+	return <div css={discountedPillStyles}>50% off</div>; //TODO confirm wording on button
```
Comment (tomrf1) [inline dotcom-rendering/src/components/marketing/epics/ThreeTierChoiceCards.tsx:170]: we could derive the copy from the `discount` value here

Comment (charleycampbell) [reply dotcom-rendering/src/components/marketing/epics/ThreeTierChoiceCards.tsx]: I've added a solution but i'm not sure it's the most elegant

**Review by tomrf1 — APPROVED** [review]
👍

**Pull request merged**